java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -95,22 +83,22 @@ class ReferenceModeStore private constructor(
     val containerStore: DirectStore<CrdtData, CrdtOperation, Any?>,
     /* internal */
     val backingKey: StorageKey,
-    backingType: Type,
-    cleanupCoroutineContext: CoroutineContext
+    backingType: Type
 ) : ActiveStore<RefModeStoreData, RefModeStoreOp, RefModeStoreOutput>(options) {
     // TODO(#5551): Consider including a hash of the storage key in log prefix.
     private val log = TaggedLog { "ReferenceModeStore" }
 
     /**
      * A queue of incoming updates from the backing store, container store, and connected proxies.
      */
-    private val receiveQueue by lazy {
-        MessageQueue(
-            handleProxyMessage,
-            handleContainerMessage,
-            handleBackingStoreMessage
```

Comment:
I suspect part of the reason why they were split into three here was to make it easier to inspect what's going on... but that doesn't seem sufficient-enough of a reason to add that much complexity, so SimpleQueue makes sense here.

Reply:
Should I do something like add the ability to provide named `Op` implementations to help with that?